theme.ts

```diff
@@ -2,7 +2,7 @@ import { extendTheme } from "@chakra-ui/react"
 
 const theme = extendTheme({
   colors: {
-    darkBlurple: '#3200AD',
+    darkBlurple: '#4000B4',
```

Comment:
can't tell if this any different lol but I think I got the right value from figma? I'm a figma newbie tbh

Reply:
ya this should be the right value